modules/core/server/views/layout.server.view.html

```diff
@@ -1,5 +1,5 @@
 ﻿<!DOCTYPE html>
-<html lang="en">
+<html lang="en" {% if env === "production" %}ng-strict-di{% endif %}>
```

Comment:
@ilanbiala right now ngAnnotate is only being used in build task for production so I kept ngStrictDi only for production otherwise it will break.